src/modules/Dashboard/features/Participant/Activities/Activities.tsx

```diff
@@ -90,14 +92,16 @@ export const Activities = () => {
   );
 
   return (
-    <ActivityGrid
-      rows={activities}
-      TakeNowModal={TakeNowModal}
-      order={'desc'}
-      orderBy={''}
-      isLoading={isLoading || isSubjectLoading}
-      data-testid={dataTestId}
-    />
+    <Box sx={{ padding: `${theme.spacing(3.2)} ${theme.spacing(1.6)}` }}>
+      <ActivityGrid
+        rows={activities}
+        TakeNowModal={TakeNowModal}
+        order="desc"
+        orderBy=""
+        isLoading={isLoading || isSubjectLoading}
+        data-testid={dataTestId}
+      />
+    </Box>
```

Comment:
Ah, you're right. I removed the padding provided by the `TabPanel` so that the Schedule tab could have the correct amount (ie, none — that tab's padding is provided by it's child content, to allow borders to extend to the edges of the tab), and re-added it here for the Activities tab. 

The padding here is equivalent to the spacing on the Schedule tab, and I assumed it was uniform across tabs. It should be updated in the [additional commit](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1680/commits/1b7bf99f4265bce8aa6aecfcd757c12d809b2736) I pushed up.

Reply:
This _does_ make me realize that the horizontal padding for the header, on both the Dashboard and Participant Details screens are off. 

In Figma, it looks like it should be a uniform 32px, except for the breadcrumbs, while in implementation it varies between 24 and 34px.